### ISSUE TITLE
Issue installing AmpScan on Python 3.7 due to vtk version

### ISSUE THREAD

**Issue by jack-parsons** [abel-research/ampscan#16]
Problem:
vtk 8.1.0 is not available for Python 3.7 using pip install

To reproduce, create a new virual conda environment with Python 3.7 with
`conda create -n test_env python=3.7`.
Then select 'y' (this will take a while to install).
Activate with `conda activate test_env`.
Now install the dependancies, as instructed on docs page with `conda install numpy scipy pyqt matplotlib vtk==8.1.0` which reveals package conflicts.
Now if you run command `pip install AmpScan` to install AmpScan, it comes up with the following errors:

>  ERROR: Could not find a version that satisfies the requirement vtk==8.1.0 (from AmpScan) (from versions: 8.1.2)

>  ERROR: No matching distribution found for vtk==8.1.0 (from AmpScan)`

**Comment by jack-parsons** [comment]
Changing the vtk==8.1.0 to vtk==8.2.0 fixes the dependency conflicts. Will test to see if anything breaks as a result.

**Comment by JoshuaSteer** [comment]
Does it work with Conda?

**Comment by jack-parsons** [comment]
I'll test installing in new environment with Conda

**Comment by jack-parsons** [comment]
vtk 8.2.0 seems to have a bug https://github.com/conda-forge/vtk-feedstock/issues/80 which means I get an import error.
> ModuleNotFoundError: No module named 'vtkRenderingFreeTypePython'
May have to just wait for new version of vtk to fix it before we can get AmpScan working on Python 3.7, as vtk 8.1.0 has requirement python<3.7

**Comment by JoshuaSteer** [comment]
If it works with pip I'm happy to just use that

**Comment by jack-parsons** [comment]
By the looks of things, vtk 8.2.0 isn't working with PyCharm right now (even with pip). The issue is still open on the vtk repo so presumably it'll be fixed at some point.

**Comment by JoshuaSteer** [comment]
At least it's being fixed, shall we just go with Python 3.6 and VTK 8.1 for the mo?

**Comment by jack-parsons** [comment]
Agreed. I've now dropped support for Python 3.7 in 597ecb5cd36aaeada2c665794ab4d1cb13e76670, so when people try to pip install it says requires 3.6. I've also added all the requirements to setup.py so all requirements are installed when `pip install Amp`  is called. This should remove the need to have this in the docs:
![image](https://user-images.githubusercontent.com/7894397/62292387-358efe80-b45e-11e9-895d-8008053e9a0a.png)
As they are installed automatically.
